bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -149,6 +149,26 @@ <h2>{{ ftl('home-mozilla-makes-privacy') }}</h2>
           ) %}
         <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button mzp-t-secondary" id="homepage-get-mozilla-vpn" data-cta-text="Get Mozilla VPN" data-cta-type="link">{{ ftl('home-cta-get-vpn') }}</a>
       {% endcall %}
+
+      {% set monitor_desc = 'Automaticly reclaim your private info' if LANG.startswith('en-') else ftl('home-cta-get-monitor')  %}
+
+      <!-- Monitor -->
```

Comment:
**Nit:** I think it would also be a good idea to add a conditional to only show this picto item if the strings for it have been translated?